Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -0,0 +1,713 @@
+    args = demisto.args()
+    af_cookie = args.get('af_cookie')
+    results, status = get_search_results('samples', af_cookie)
+    md = tableToMarkdown(f'Search Samples Results is {status}', results)
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['text'],
+        'Contents': results,
+        'EntryContext': {'AutoFocus.SamplesResults(val.ID == obj.ID)': results},
+        'HumanReadable': md
+    })
+
+
+def sessions_search_results_command():
+    args = demisto.args()
+    af_cookie = args.get('af_cookie')
+    results, status = get_search_results('sessions', af_cookie)
+    md = tableToMarkdown(f'Search Sessions Results is {status}:', results)
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['text'],
+        'Contents': results,
+        'EntryContext': {'AutoFocus.SessionsResults(val.ID == obj.ID)': results},
+        'HumanReadable': md
+    })
+
+
+def get_session_details_command():
+    args = demisto.args()
+    session_id = args.get('session_id')
+    result = get_session_details(session_id)
+    md = tableToMarkdown(f'Session {session_id}:', result)
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['text'],
+        'Contents': result,
+        'EntryContext': {'AutoFocus.Sessions(val.ID == obj.ID)': result},
+        'HumanReadable': md
+    })
+
+
+def sample_analysis_command():
+    args = demisto.args()
+    sample_id = args.get('sample_id')
+    os = args.get('os')
+    filter_data = False if args.get('filter_data') == 'False' else True
+    analysis = sample_analysis(sample_id, os, filter_data)
+    context = createContext(analysis, keyTransform=string_to_context_key)
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['text'],
+        'Contents': {'ID': sample_id, 'Analysis': analysis},
+        'HumanReadable': f'### Sample Analysis results for {sample_id}:',
+        'EntryContext': {f'AutoFocus.SampleAnalysis(val.ID == obj.ID)': {'ID': sample_id, 'Analysis': context}},
+    })
+    for category_name, category_data in analysis.items():
+        print_hr_by_category(category_name, category_data)
+
+
+def tag_details_command():
+    args = demisto.args()
+    tag_name = args.get('tag_name')
+    result = autofocus_tag_details(tag_name)
+    md = tableToMarkdown(f'Tag {tag_name} details:', result, headerTransform=string_to_table_header)
+    context = createContext(result, keyTransform=string_to_context_key)
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['text'],
+        'Contents': result,
+        'EntryContext': {'AutoFocus.Tag(val.ID == obj.ID)': context},
+        'HumanReadable': md
+    })
+
+
+def top_tags_search_command():
+    args = demisto.args()
+    scope = args.get('scope')
+    tag_class = args.get('class')
+    private = True if args.get('private') == 'True' else False
```

Comment:
you only need to write `args.get('private') == 'True'`
the same with the 3 following lines.

Reply:
- [x] Check